src/main/java/uk/gov/ida/verifylocalmatchingserviceexample/model/Person.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.ida.verifylocalmatchingserviceexample.model;
+
+import org.jdbi.v3.core.mapper.reflect.ColumnName;
+
+import java.time.LocalDate;
+
+
+public class Person {
+    private String surname;
+    @ColumnName("date_of_birth")
+    private LocalDate dateOfBirth;
+
+    //required by jdbi mapper
+    public Person() {}
+
+    public Person(String surname, LocalDate dateOfBirth) {
+        this.surname = surname;
+        this.dateOfBirth = dateOfBirth;
+    }
+
+    public String getSurname() {
+        return surname;
+    }
+
+    public LocalDate getDateOfBirth() {
+        return dateOfBirth;
+    }
+
+    public void setSurname(String surname) {
+        this.surname = surname;
+    }
+
```

Comment:
setters are required by JDBI. If someone knows how we can avoid, it would be great!

Reply:
might be worth looking at this branch of stub-idp where I used JDBI, don't think that needed setters... https://github.com/alphagov/ida-stub-idp/compare/bau/use_jdbi

Reply:
I will have a look at it. Thanks!